packages/server/lib/crons/exportMetrics.ts

```diff
@@ -0,0 +1,46 @@
+import * as cron from 'node-cron';
+import { errorManager, ErrorSourceEnum, connectionService } from '@nangohq/shared';
+import { stringifyError, getLogger, metrics } from '@nangohq/utils';
+import tracer from 'dd-trace';
+
+const logger = getLogger('Server');
+const cronName = '[exportUsageMetrics]';
+const cronMinutes = 5;
```

Comment:
happy to discuss. it is doing a single query to the main db so it is definitely not a very heavy cron